test/rdoc/rdoc_markup_to_html_crossref_test.rb

```diff
@@ -33,6 +33,33 @@ def test_convert_CROSSREF
     assert_equal para("<code># :stopdoc:</code>:"), result
   end
 
+  def test_convert_CROSSREF_backslash_in_tt
+    hyperlink_all = @options.hyperlink_all
+    @options.hyperlink_all = false
+
+    formatter = RDoc::Markup::ToHtmlCrossref.new @options, 'C9.html', @c9_b
+    result = formatter.convert '<tt>C1</tt>'
+
+    assert_equal para('<a href="C1.html"><code>C1</code></a>'), result
+
+    formatter = RDoc::Markup::ToHtmlCrossref.new @options, 'C9.html', @c9_b
+    result = formatter.convert '<tt>.bar.hello()</tt>'
+
+    assert_equal para('<code>.bar.hello()</code>'), result
+
+    formatter = RDoc::Markup::ToHtmlCrossref.new @options, 'C9.html', @c9_b
+    result = formatter.convert '<tt>.bar.hello(\)</tt>'
+
+    assert_equal para('<code>.bar.hello(\)</code>'), result
+
+    formatter = RDoc::Markup::ToHtmlCrossref.new @options, 'C9.html', @c9_b
+    result = formatter.convert '<tt>.bar.hello(\\\\)</tt>'
+
+    assert_equal para('<code>.bar.hello(\\)</code>'), result
+  ensure
+    @options.hyperlink_all = hyperlink_all
```

Comment:
It seems that we don't need this because we always set `@options.hyperlink_all` in `setup`.